src/demo-app/select/select-demo.html

```diff
@@ -54,3 +54,27 @@
 
 </div>
 <div style="height: 500px">This div is for testing scrolled selects.</div>
+
+<md-card>
```

Comment:
I don't think we need this example. The examples in the demo app are there to be able to click around the main use cases, they're not for actual testing.